src/frontend/screens/DownloadManager/components/DownloadManagerItem/index.tsx

```diff
@@ -60,7 +60,7 @@ const DownloadManagerItem = ({ element, current }: Props) => {
       }
     }
     getNewInfo()
-  }, [])
+  }, [element])
```

Comment:
we need this so when the current element changes to the next in the list, it fetches the new gameInfo